### PR TITLE
Canan

### PR DESCRIPTION
**Pull request by JaneRuthMartin**
Hi Jordan,

I updated my page and test classes. 

Thanks for merging them.

### CODE REVIEW

File: src/test/java/com/automation/pages/activityStream/EventPage.java [hunk 1]
```diff
@@ -7,16 +7,47 @@
...
 public class EventPage extends AbstractPageBase {
+    // Tester Canan
 
     // Upload elements
     @FindBy (id = "bx-b-uploadfile-blogPostForm_calendar")
-    protected WebElement UploadFilesBtn;
+    protected WebElement uploadFilesIcon;
+
+    @FindBy (name = "bxu_files[]")
+    protected WebElement uploadFilesAndImages;
```
Comment (jlhankison) [inline src/test/java/com/automation/pages/activityStream/EventPage.java:17]: This locator already exists, is it pointing to the same location as is already in the repository?
`   @FindBy(name = "bxu_files[]")	
    private WebElement uploadfilesandimages;`

File: src/test/java/com/automation/pages/activityStream/EventPage.java [hunk 2]
```diff
@@ -7,16 +7,47 @@
...
+    protected WebElement recentItems;
+
+    @FindBy (xpath = "//div[@class='bx-file-dialog-content-wrap']//div//span//a")
+    protected WebElement firstItem;
+
+    @FindBy (xpath = "//span[text()='My Drive']")
+    protected WebElement myDrive;
+
+    @FindBy (xpath = "//a[text()='Uploaded files']")
+    protected WebElement uploadedFiles;
```
Comment (jlhankison) [inline src/test/java/com/automation/pages/activityStream/EventPage.java:35]: This WebElement name also already exists, again is it linking to the same location?
`@FindBy(name = "bxu_files[]")	
    private WebElement uploadfilesandimages;`

**Review by jlhankison — APPROVED** [review]
I'll approve your code but it looks like you made some changes to some locators, can you check my comments and let me know if that is correct?

File: src/test/java/com/automation/pages/activityStream/EventPage.java (hunk 1 above)
Comment (JaneRuthMartin) [reply src/test/java/com/automation/pages/activityStream/EventPage.java:17]: Yes, it is pointing the same. I did not recognize that it was already there.

File: src/test/java/com/automation/pages/activityStream/EventPage.java (hunk 2 above)
Comment (JaneRuthMartin) [reply src/test/java/com/automation/pages/activityStream/EventPage.java:35]: Yes, it is pointing the same. I did not recognize that it was already there.

**Pull request merged**